scalastyle-config.xml

```diff
@@ -0,0 +1,117 @@
+<scalastyle>
+    <name>Scalastyle standard configuration</name>
+    <check level="warning" class="org.scalastyle.file.FileTabChecker" enabled="true"></check>
+    <check level="warning" class="org.scalastyle.file.FileLengthChecker" enabled="true">
+        <parameters>
+            <parameter name="maxFileLength"><![CDATA[800]]></parameter>
+        </parameters>
+    </check>
+    <check level="warning" class="org.scalastyle.file.HeaderMatchesChecker" enabled="true">
+        <parameters>
+            <parameter name="header"><![CDATA[/*
+ * Copyright 2019 HM Revenue & Customs
```

Comment:
Could this be looked at? Copyright notice really shouldn't be in the middle of the xml like this...

(Update: I've noticed it's in the frontend like this as well, ideally should be removed but can have a chat with Duane about that as he's doing frontend changes atm)